addon/ng2/blueprints/ng2/files/__path__/vendor.ts

```diff
@@ -0,0 +1,17 @@
+// Typescript emit helpers polyfill
+import 'ts-helpers';
+
+import '@angular/core';
+import '@angular/common';
+import '@angular/compiler';
+import '@angular/http';
+import '@angular/router';
+import '@angular/platform-browser';
+import '@angular/platform-browser-dynamic';
+
+<% if(isMobile) { %>
```

Comment:
It's not dynamic, it's a template file. A project generated with `ng new --mobile` will have this line, otherwise it won't.